### DISCUSSION TITLE
webhook from the outside not working

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2152] [Help] [answered]
Hello ;)

I did a simple webhook that trigger a discord message. When I test is with the test button it is working but when I trigger the webhook from the outside (n8n) it is not working...

Thanks a lot

**Comment by shogunpurple** [comment, marked as answer]
Hello @oswaldB 

Can you provide a little more information? It would be good to see your automation and the way it is set up, and we can debug from there. 

Is it possibly a network issue? Do N8N and budibase have the relevant network access to each other?

**Reply by oswaldB** [reply]
Sorry it works fine.

I just understand that I need to trigger the schema post webhook first and then the trigger one.

Thank you
